### DISCUSSION TITLE
inconsistent start menu search position

### DISCUSSION THREAD

**Discussion by MikuDroid** [valinet/ExplorerPatcher#251] [Q&A] [answered]
after connecting / disconnecting multiple monitor, the search bar position changed. is this known issue? a quick explorer restart fixed it temporarily tho. After i did the same thing, same issue happened again (ofc duh). i'm using 22000.282.32.0 btw.

https://user-images.githubusercontent.com/22476406/139646074-aa60b5e9-5b35-4391-b906-2597f2a211d0.mp4

**Comment by valinet** [comment, marked as answer]
Known or unknown, I don’t know how to fix that anyway. I noticed indeed that the search panel sometimes forgets where it was supposed to go. I don’t know why. Its operation is overly complicated, as is the case with most of the recent stuff in Windows.
